lib/fluent/plugin/in_heartbeat.rb

```diff
@@ -30,7 +34,18 @@ def start
             "format" => "json",
             "level" => "info"
           }
-        elsif
+        elsif mdm
+          record = {
+            "Namespace"=>@namespace,
+            "Metric"=>@metric,
+            "Dimensions"=> {
+              "ColoRegion" => coloRegion,
+              "DeploymentInstance" => coloId,
+              "BuildVersion" => buildVersion
```

Comment:
Tested on INT yet?

Reply:
Yes, of course. Beautiful nice graph with colors and fanciness.